sqsjobs/item.go

```diff
@@ -28,7 +28,7 @@ const (
 	BinaryType              string = "Binary"
 	ApproximateReceiveCount string = "ApproximateReceiveCount"
 	fifoSuffix              string = ".fifo"
-	jobAlreadyStopped       string = "Failed to acknowledge the job. The pipeline is probably stopped."
+	pipelineStoppedError    string = "Failed to ACK/NACK the job. The pipeline is probably stopped."
```

Comment:
Should probably be `Failed to ACK/NACK or requeue the job. The pipeline is probably stopped` as it is also used for failed requeue.